projects/admin/src/app/record/circulation-logs/circulation-log/circulation-log.component.html

```diff
@@ -52,11 +55,9 @@
 </div>
 <div id="circulation-history-{{ record.metadata.pid }}" [collapse]="isCollapsed" [isAnimated]="true" class="col-lg-12 pl-3">
   <dl class="row pl-1">
-    <dt class="col-2 mb-0" translate>Loan status</dt>
-    <dd class="col-10 mb-0 pl-1">{{ record.metadata.record.state | translate }}</dd>
     <dt class="col-2 mb-0" translate>Pickup location</dt>
-    <dd class="col-10 mb-0 pl-1">{{ record.metadata.record.pickup_location_pid | getRecord : 'locations' : 'field': 'name' | async }}</dd>
+    <dd class="col-10 mb-0 pl-1">{{ record.metadata.loan.pickup_location.name | ucfirst }}</dd>
     <dt class="col-2 mb-0" translate>Operator</dt>
-    <dd class="col-10 mb-0 pl-1">{{ record.metadata.user_name }}</dd>
+    <dd class="col-10 mb-0 pl-1">{{ record.metadata.user_name | ucfirst }}</dd>
```

Comment:
Why `ucfirst` ? a username could be case-sensitive. If we copy/paste this informations, this coud be source oof problem.

Reply:
I put that in to always have a capital letter at the beginning. For example "system" is in lower case. But I can remove it.